src/shared/directives/substitutor/services/placeholder.service.spec.ts

```diff
@@ -110,7 +110,7 @@ and markdown is \${Markdownlabel} and address is \${Address} and document \${D8D
     });
 
     it('should substitute fields with multi select values', () => {
-      let pageFormFields = { '_1_one': ['code1', 'code2'], '_1_one-LABEL': ['label1', 'label2'], '_3_three': 'simpleValue' };
+      let pageFormFields = { '_1_one': ['code1', 'code2'], '_1_one---LABEL': ['label1', 'label2'], '_3_three': 'simpleValue' };
```

Comment:
@paul-graham - I was wondering whether to `let` you off on this one, but seeing as you were already changing this specific line...

Reply:
Change pushed. :)